README.md

```diff
@@ -1 +1,41 @@
-# reference-libraries
\ No newline at end of file
+# Team 1540 Reference Libraries
+
+A common library of useful classes and systems intended to be used for all Team 1540 robots.
+
+## Installation
+
+### Using Gradle
+Add the library by adding these lines in your `build.gradle` file:
+
+```Gradle
+repositories {
+     (other repos)
+     maven { url 'https://jitpack.io' }
+}
+
+dependencies {
+     (other dependencies)
+     compile 'com.github.flamingchickens1540:reference-libraries:-SNAPSHOT'
```

Comment:
Jitpack needs the branch name too. Change to
`compile 'com.github.flamingchickens1540:reference-libraries:gradle-integration-SNAPSHOT'`

Reply:
Good catch, fixing now. That said, when we actually merge this it'll be based off releases so it'll be more like `org.team1540:reference-libraries:1.0` (yes, `org.team1540`, see my comment below)